helm/g2p-sandbox-fynarfin-SIT/values.yaml

```diff
@@ -31,7 +31,13 @@ ph-ee-g2psandbox:
       
     ph_ee_connector_ams_mifos:
       enabled: true
-      imageTag: v1.2.2 
+      imageTag: v1.2.2
+      ams_local_interop_host: "https://fynams.sandbox.fynarfin.io/"
```

Comment:
can you check fin12 profile in the config map? these changes might be available there as well.

Reply:
I have checked Phee config map, they are not there.

Reply:
this should be added there. ref:  https://github.com/openMF/ph-ee-connector-ams-mifos/blob/master/src/main/resources/application-fin12.yml